reference/6/About/about_Special_Characters.md

```diff
@@ -112,6 +133,20 @@ The output from this command is:
 
 Column1         Column2         Column3
 
+UNICODE CHARACTER (`u{x})
+The Unicode escape sequence allows you to specify any Unicode character
+including Unicode surrogate characters often used for Emoji e.g. `u{1F44D}.
```

Comment:
Perhaps just say "including symbol characters used for emoji, e.g. `` `u{1F44D}. ``"? (either way, "emoji" should be lowercase) ([`U+1F44D`](http://www.fileformat.info/info/unicode/char/1f44d/index.htm) is not a _surrogate character_, it is a character that _happens to be outside the BMP_ (a code point > 0xFFFF); its code-point type is _Graphic_, and its category is _[so] (Symbol, other)_. In fact, there is no such thing as a surrogate _character_, only surrogate _code points_ that _in pairs_ form an _assigned character_.